spamcan.py

```diff
@@ -17,15 +19,19 @@
 
 bottle.debug(True)
```

Comment:
Since a lot of backend stuff will probably change (e.x sqlite) we can leave this as is for the time being?